internal/p2p/peermanager.go

```diff
@@ -38,11 +38,18 @@ const (
 	PeerStatusBad  PeerStatus = "bad"  // peer observed as bad
 )
 
+type peerConnectionDirection int
```

Comment:
fair, though I wanted to avoid having a meaningful 0 value in the map, but I guess iota starts at 0, so I missed that. added an invalid value at the beginning.